src/views/ThresholdUpdates.vue

```diff
@@ -253,7 +253,7 @@
         </section>
 
         <aside class="desktop-only" v-show="segmentSelected === 'pending' && currentJob">
-          <JobConfiguration :title="title" :job="currentJob" :productCount="productCount" :status="currentJobStatus" :type="freqType" :key="currentJob"/>
+          <JobConfiguration :title="title" :job="currentJob" :productCount="productCount" :status="currentJobStatus" :type="freqType" :key="currentJob" :jobEnums="jobEnums" />
```

Comment:
I think this may not be needed as we are not using jobEnums in JobConfiguration component